Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -330,33 +330,35 @@ final class IfconfigExprTests: XCTestCase {
       #endif
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected value in function call", fixIts: ["insert value"])
+        DiagnosticSpec(message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion")
       ]
     )
   }
 
   func testIfconfigExpr24() {
     assertParse(
       """
-      #if canImport(A, 2.2)
+      #if canImport(A, 1️⃣2.2)
         let a = 1
       #endif
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: 2nd parameter of canImport should be labeled as _version or _underlyingVersion
+        DiagnosticSpec(message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion")
       ]
     )
   }
 
   func testIfconfigExpr25() {
     assertParse(
       """
-      #if canImport(A, 2.2, 1.1)
+      #if canImport(A, 1️⃣2.22️⃣, 1.1)
         let a = 1
       #endif
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: canImport can take only two parameters
+        DiagnosticSpec(locationMarker: "1️⃣", message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion"),
+        // TODO: Old parse produce "canImport can take only two parameters" here
+        DiagnosticSpec(locationMarker: "2️⃣", message: "trailing components of version code '2.2' are ignored", severity: .warning),
```

Comment:
In my implementation, I couldn't find a clean workaround to determine whether the trailing components are extra parameters here. Do you have any idea? Or could we just use the diagnostic message here?